src/main/java/bio/terra/common/exception/CommonExceptions.java

```diff
@@ -0,0 +1,9 @@
+package bio.terra.common.exception;
+
+/** Common exceptions that tend to be thrown in different areas */
+public class CommonExceptions {
+
+  public static final FeatureNotImplementedException TRANSACTIONS_NOT_IMPLEMENTED_IN_AZURE =
```

Comment:
What is the benefit of doing this vs having an exception subclass that's thrown instead? E.g.
```
public class TransactionSupportException extends FeatureNotImplementedException {
  public TransactionSupportException() {
      super("Transaction support is not yet implemented for Azure backed datasets");
  }
}
[...]

    throw new TransactionSupportException();

```

The subclass gives you the same code sharing feature, and doesn't require a static value which the program may never use.

Reply:
I was doing this as a potential solution to use for different types of classes where there may be repeated messages (not just this case...we'd need to fill out). For instance, we have several instances of PdaoException that get used a lot..it'd be nice to have that captured in constants

Reply:
I'd need to see the specific cases where a `PdaoException` is repeated to understand what problem this would be solving. In those case I would most likely prefer either a specific exception subclass or constructor. In this case though I don't see how using a constant object is better than a subclass.

In general, while exceptions are objects like any other, I think it's better to only have exceptions created and used during an active `throw` operation.

Reply:
I guess...though a base exception seems not unreasonable to have as a constant.  But like, Pdao exceptions that throw with the message `"create staging load history table failed for..` appear repeatedly or `""Invalid row ids supplied"`